scripts/Muon/GUI/ElementalAnalysis/PeriodicTable/periodic_table.py

```diff
@@ -0,0 +1,763 @@
+from collections import OrderedDict
+import logging
+from PyQt4 import QtCore
+from PyQt4 import QtGui as qt
+
```

Comment:
I think we should add a few lines at the top just saying that this creates a peridoic table with colouring and buttons for each element